src/Arcus.Messaging.Pumps.ServiceBus/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -34,9 +34,28 @@ public class AzureServiceBusMessagePumpOptions
         /// <remarks>When turned off, clients have to explicitly mark the messages as completed</remarks>
         public bool AutoComplete { get; set; }
 
+        /// <summary>
+        /// Gets or sets the unique identifier for this background job to distinguish this job instance in a multi-instance deployment.
+        /// </summary>
+        public string JobId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the value indicating whether or not a new Azure Service Bus Topic subscription has to be created when the <see cref="AzureServiceBusMessagePump{TMessage}"/> starts.
+        /// The subscription will be deleted afterwards when the message pump stops if the options <see cref="TopicSubscription.DeleteOnStop"/> is selected.
+        /// </summary>
+        /// <remarks>
+        ///     Provides capability to create and delete these subscriptions. This requires 'Manage' permissions on the Azure Service Bus Topic or namespace.
+        /// </remarks>
+        public TopicSubscription TopicSubscription { get; set; }
+
         /// <summary>
         ///     Default settings
         /// </summary>
-        internal static AzureServiceBusMessagePumpOptions Default => new AzureServiceBusMessagePumpOptions { AutoComplete = true };
+        internal static AzureServiceBusMessagePumpOptions Default => new AzureServiceBusMessagePumpOptions
+        {
+            AutoComplete = true,
+            JobId = Guid.NewGuid().ToString(),
+            TopicSubscription = TopicSubscription.CreateOnStart
```

Comment:
Didn't we decide to clean up after finished like you've suggested? If so, please don't forget to update docs with new default

Reply:
I suggested it, but wasn't sure you agreed 😄 .

Reply:
It's a fair suggestion so go ahead! 👍